eos-paygd/meson.build

```
@@ -36,6 +36,10 @@ systemdsystemunitdir = get_option('systemdsystemunitdir')
 if systemdsystemunitdir == ''
   systemdsystemunitdir = systemd.get_pkgconfig_variable('systemdsystemunitdir')
 endif
+tmpfilesdir = get_option('tmpfilesdir')
+if tmpfilesdir == ''
+  tmpfilesdir = systemd.get_pkgconfig_variable('tmpfilesdir')
+endif
```

Try this:

```suggestion
tmpfilesdir = get_option('tmpfilesdir')
if tmpfilesdir == ''
  tmpfilesdir = systemd.get_pkgconfig_variable('tmpfilesdir', define_variable: ['prefix', prefix])
endif
```

Found at https://gitlab.freedesktop.org/pipewire/pipewire/-/merge_requests/235/diffs